### DISCUSSION TITLE
Welcome to Towny Discussions!

### DISCUSSION THREAD

**Discussion by LlmDl** [TownyAdvanced/Towny#4550] [General]
## 👋 Welcome!
  We’re ~~using~~ going to try out Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Depending on the usefulness of this board, it may or may not end up replacing some of the feature-requests aspect of the issue tracker. 
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Momshroom** [comment]
Hi, I’m Momshroom.  I run a smallish towny server, try out the pre-releases, act as a bellwether for MySQL issues, report bugs,  and am working on some PRs.

**Reply by TheBeeCollector** [reply]
@LlmDl Thanks for giving this a shot! Can we get a Q/A category for questions?

**Reply by LlmDl** [reply]
I believe all the categories are there to choose from when you make a new discussion/thread. Let me know if that's not automatic.

**Reply by TheBeeCollector** [reply]
Yeah I see them now

**Comment by TheBeeCollector** [comment]
My name is TheBeeCollector, and I run a towny server in my local community, I also dabble a bit in plugin development.

**Comment by SavvaKulikov** [comment]
Hello, I've sponsored the Towny and would like to get TownyCamp and Eventwar. Can you send it, please?

**Reply by LlmDl** [reply]
> Hello, I've sponsored the Towny and would like to get TownyCamp and Eventwar. Can you send it, please?

Hello @SavvaKulikov thank you for sponsoring, please join the discord using this link where you will be able to download the plugins: https://discord.gg/gnpVs5m
